lib/ts/recipe/session/with-jwt/recipeImplementation.ts

```diff
@@ -110,12 +124,11 @@ export default function (
             let newSession = await originalImplementation.refreshSession({ req, res });
             let accessTokenPayload = newSession.getAccessTokenPayload();
 
-            // Remove the old jwt
-            if (config.jwt.getPropertyNameFromAccessTokenPayload !== undefined) {
-                let jwtPropertyName = config.jwt.getPropertyNameFromAccessTokenPayload(accessTokenPayload);
+            let jwtPropertyName = accessTokenPayload[ACCESS_TOKEN_PAYLOAD_JWT_PROPERTY_NAME_KEY];
+
+            // Remove the old JWT
+            if (jwtPropertyName !== undefined) {
                 delete accessTokenPayload[jwtPropertyName];
```

Comment:
Delete _jwtpName as well?

Reply:
Logic has been refactored to always remove the old jwt and property name before creating a new JWT